packages/paste-core/primitives/box/src/types.ts

```diff
@@ -90,6 +90,7 @@ export interface BoxElementProps extends Omit<React.HTMLAttributes<HTMLElement>,
   element?: string;
   /** variant for variant styling */
   variant?: string;
+  disabled?: boolean;
```

Comment:
This should be mentioned in a changeset too. Box should be able to accept the disabled prop if used as an input type or button element

Reply:
kind of amazed this wasn't already there. 🤯